resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -127,6 +127,11 @@
                     <div class="col-5"><b>TS3 UID: <a class="info-tooltip" href="#" data-toggle="tooltip" title="In TeamSpeak 3, go to Tools->Identifies. Paste your 'Unique ID' here for bot integration"><i class="fas fa-info-circle"></i></a></b></div>
                     <div class="col-7">{{ html()->text('ts3', Auth::user()->ts3)->class(['form-control']) }}</div>
                 </div>
+                
```

Comment:
Remove this user input - no longer needed since we're pulling the UID from the API.